<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的摄像头</string>
  </property>
  <widget class="QFrame" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
    <item row="0" column="0">
     <widget class="QVideoWidget" name="widgetCamera" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>摄像头：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_CameraName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>分辨率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_Resolution"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>最大帧率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_MaxFrameRate"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSnap">
        <property name="text">
         <string>单张抓拍</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>间隔时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVedio">
        <property name="text">
         <string>连续抓拍</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
